testcases/multizone_office_complex_air/models/MultizoneOfficeComplexAir/BaseClasses/HVACSide/HVAC.mo

```diff
@@ -163,12 +163,12 @@ model HVAC "Full HVAC system that contains the air side and water side systems"
     annotation (Placement(transformation(extent={{70,-36},{90,-14}})));
 equation
   connect(chiWatNet.ports_a[1], floor1.port_b_CooWat) annotation (Line(
-      points={{40,-91.1333},{40,-94},{106,-94},{106,6},{130.563,6},{130.563,20}},
```

Comment:
Please remove unnecessary changes to the annotations in the PR if possible

Reply:
Some annotations were unintentionally changed when opening a model in Dymola interface. I will pay attention to these kind of annotations.